packages/components/src/index.native.js

```diff
@@ -75,6 +76,7 @@ export { default as Gradient } from './mobile/gradient';
 export { default as ColorSettings } from './mobile/color-settings';
 export { LinkPicker } from './mobile/link-picker';
 export { default as LinkSettings } from './mobile/link-settings';
+export { default as SegmentedControl } from './mobile/segmented-control';
```

Comment:
Surprisingly this component was not exported  I saw that it was only used in another component).